### DISCUSSION TITLE
播放alist挂载网盘视频较慢

### DISCUSSION THREAD

**Discussion by TeMouRen** [hooke007/mpv_PlayKit#412] [回收]
与potplayer对比，mpv lazy的下载速度比potplayer慢

**Comment by ZhuMing19920416** [comment]
麻烦问一下是怎么用MPV挂在alist的？

**Comment by Lonely233233** [comment]
> 与potplayer对比，mpv lazy的下载速度比potplayer慢

不觉得MPV挂载alist速度慢 我觉得MPV挂载alist快很多
![image](https://github.com/hooke007/MPV_lazy/assets/106970209/cb97622d-21a3-40f7-8fb7-781b4a10b5be)

**Comment by hooke007** [comment]
我倾向于是个人的网络环境配置问题

**Comment by hooke007** [comment]
ALIST 爆雷，关闭问题并锁定
